bar/bootstrap.sh

```diff
@@ -1,9 +1,19 @@
 #!/usr/bin/bash
-sudo yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
-sudo yum-config-manager --enable docker-ce-nightly
-sudo yum-config-manager --enable docker-ce-test
-sudo yum -y install docker-ce docker-ce-cli containerd.io
-sudo yum install -y java-1.8.0-openjdk-devel
-curl --silent --location http://pkg.jenkins-ci.org/redhat-stable/jenkins.repo | sudo tee /etc/yum.repos.d/jenkins.repo
-sudo rpm --import https://jenkins-ci.org/redhat/jenkins-ci.org.key
-sudo yum -y install jenkins
+sudo yum -y install git
+ssh-keygen -t rsa -b 4096
+eval "$(ssh-agent -s)"
+ssh-add ~/.ssh/id_rsa
+ssh -T git@github.com
+sudo yum install -q -y yum-utils
+sudo yum-config-manager \
+    --add-repo \
+    https://download.docker.com/linux/centos/docker-ce.repo
+sudo yum -y install docker docker-ce docker-ce-cli containerd.io
```

Comment:
No need to inline `sudo` commands.
Make sure that the user has root privileges and add nice comment at the start of the script that saying that "this scripts need's to run as root"

Reply:
fixed